assets/queries/ansible/aws/api_gateway_public_lambda/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "Public_Lambda_via_API_Gateway",
```

Comment:
Change the id to lowercase letters: `public_lambda_via_api_gateway`
Also, change the directory name to be equal to the query's id

Reply:
Don't forget to change the directory name to be equal to the query's id